telegram/bot.py

```diff
@@ -1573,6 +1583,32 @@ def answer_inline_query(self,
         """
         url = '{}/answerInlineQuery'.format(self.base_url)
 
+        if current_offset is not None and next_offset is not None:
+            raise ValueError('`current_offset` and `next_offset` are mutually exclusive!')
+
+        if current_offset is not None:
+            if current_offset == '':
+                current_offset = 0
+            else:
+                current_offset = int(current_offset)
```

Comment:
In what case would we have a negative value here?

Reply:
Defensive coding

Reply:
IMHO it would just make debugging very hard, if invalid negative input lead to completely unexpected behaivour instead of leading to an exception thrown … I'll add a warning, that `current_offset` shouldn't usually be passed manually bot `InlineQuery.answer(, auto_pagination=True)` should take care of it.